### ISSUE TITLE
"basename: illegal option -- f" error when using with Tmux

### ISSUE THREAD

**Issue by rstacruz** [franciscolourenco/done#127]
I'm running into an odd error where this error appears when using fish with Tmux. It's almost always reproduceable when ran with neovim, but can often appear when just using tmux.

```
~ > sleep 4
basename: illegal option -- f
usage: basename string [suffix]
       basename [-a] [-s suffix] string [...]
```

How to reproduce:

```sh
tmux # open tmux
nvim -u NONE '+:terminal' # open neovim with a terminal
sleep 4
```

Environment info:

- os: MacOS 11.6
- shell: fish 3.3.1
- tmux 3.2a
- Neovim 0.5.1
- terminal: Hyper

Thank you for all your hard work with `done`!

**Comment by rstacruz** [comment]
From what I can tell, this happens here:

```sh
    while set tmux_fish_ppid (ps -o ppid= -p $tmux_fish_pid | string trim)
        and ! string match -q "tmux*" (basename (ps -o command= -p $tmux_fish_ppid)) #<==
        set tmux_fish_pid $tmux_fish_ppid
    end
```

because `tmux_fish_ppid` is not available, the `ps` command gets this error

```
ps: option requires an argument -- p
```

Changing it to a hardcoded `fish` makes things work:

```diff
    while set tmux_fish_ppid (ps -o ppid= -p $tmux_fish_pid | string trim)
-       and ! string match -q "tmux*" (basename (ps -o command= -p $tmux_fish_ppid))
+       and ! string match -q "tmux*" fish
        set tmux_fish_pid $tmux_fish_ppid
    end
```

**Comment by ovikk13** [comment]
This is solved by this pull request: https://github.com/franciscolourenco/done/pull/124

**Comment by blissdev** [comment]
I'm also seeing this issue. Not sure why, but it just started appearing for me.

**Comment by franciscolourenco** [comment]
Fixed by #124. Please reopen if needed.
